tests/unit_tests/test_wave_07.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+#@pytest.mark.skip
+def test_swap_best_by_category():
```

Comment:
👍 Nice test for your optional swap by newest behavior.

However, the name should be updated to reflect that it's looking at newest, not best.